tests/mathematics.rs

```diff
@@ -0,0 +1,116 @@
+use opentype::mathematics::Mathematics;
+use opentype::layout::Coverage;
+use truetype::Value;
+
+use std::io::{Seek, SeekFrom};
+use std::fs::File;
+
+macro_rules! read_math_table {
+    (XITS) => {
+        {
+            let mut file = ok!(File::open("tests/fixtures/XITS-Math.otf"));
+            ok!(file.seek(SeekFrom::Start(496284)));
+            ok!(Value::read(&mut file))
+        }
+    };
+
+    (LatinModern) => {
```

Comment:
I use this for personal testing while developing, but it's probably not required for maintaining the code.